src/plans/ID3MPlan.sol

```diff
@@ -16,11 +16,28 @@
 
 pragma solidity 0.6.12;
 
+/**
+    @title D3M Plan Interface
+    @notice Plan contracts are the contract that the Hub uses to determine how much to change its position
+*/
 interface ID3MPlan {
     event Disable();
 
-    // --- Plan Functions ---
+    /**
+        @notice Determines what the position should be based on current assets and the
+        custom plan rules.
+        @param currentAssets asset balance from a specific pool in Dai [wad] denomination
+        @return uint256 target assets the Hub should wind or unwind to. Dai [wad] denomination
+    */
     function getTargetAssets(uint256 currentAssets) external view returns (uint256);
+
+    /// @notice Reports whether the plan is active
     function active() external view returns (bool);
+
+    /**
+        @notice Disables the plan so that it would instruct the Hub to unwind its entire position.
+        @dev It is up to the specific implentation exactly what this means and if it is reversible
+        base implementation is permissionless.
```

Comment:
There is no base implementation now, better say something like "Implementation should be permissioned".